rosbag2_py/src/rosbag2_py/py_utils.hpp

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef ROSBAG2_PY__PY_UTILS_HPP_
```

Comment:
nit: this whole package is already "py" - i might rename this file something more informative like `plugin_utils` - or go more generic to just `utils` (but probably best not to). The "py" part suggests that it might have something to do with interacting with the Python language or interpreter, but it doesn't

On a second note - this function would be useful for pure-C++ users as well, and is not specific to Python. Could this live comfortably in `rosbag2_cpp`?

Reply:
Yeah I'll rename it `plugin_utils.hpp`. Definitely seems like it's general enough to have broader utility in the future, so I'll add it to `rosbag2_cpp`.